juniper_official/routing/check-ospf3-io-statistics.rule

```diff
@@ -0,0 +1,160 @@
+/*
+ * Detects Osp3fIoStatsTable IO errors and notifies when anomalies are found.
+ */
+ healthbot {
+    topic routing.ospf {
+        rule check-ospf3-io-statistics {
+            synopsis "OSPF3 IO statistics analyzer";
+            description "Monitors OSPF's I/O statistics";
```

Comment:
OSPF3?

Reply:
to support ipv6 for ospf :

Below cli cmds, we need to use ospf3 for ipv6
show ospf3 statistics 
show ospf3 io-statistics

Reply:
I meant to change change OSPF to OSPF3 in description